packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -326,6 +326,18 @@ export default class ElectronAppWrapper {
 			}
 		});
 
+		ipcMain.on('open-link', async (_event, url) => {
+			try {
+				await shell.openExternal(url);
+			} catch (error) {
+				console.error(`Failed to open URL: ${url}`, error);
+			}
+		});
```

Comment:
Consider using [`bridge().openExternal`](https://github.com/laurent22/joplin/blob/786e58ed544482077fd09048b9f84f44e9ea1f1b/packages/app-desktop/bridge.ts#L390) here (importing `bridge` from `packages/app-desktop/bridge.ts` since this is in the main process) rather than `shell.openExternal`. Depending on the source of `url`, there are [security issues](https://www.electronjs.org/docs/latest/tutorial/security#15-do-not-use-shellopenexternal-with-untrusted-content) with calling `openExternal` directly.

Alternatively, depending on where `open-link` is used, [`shim.openUrl`](https://github.com/laurent22/joplin/blob/786e58ed544482077fd09048b9f84f44e9ea1f1b/packages/lib/shim.ts#L349) could be used instead.